gradle/publish.gradle

```diff
@@ -66,10 +68,35 @@ task prepareGhCredentials {
 prepareGhPages.dependsOn groovydoc
 publishGhPages.dependsOn prepareGhCredentials
 
+task prepareGradlePluginsRepoRelease {
+    description = "Prepare the release to the Gradle Plugins Repository"
+    group = 'release'
```

Comment:
`nit`: When `group` is set, it shows up in the IDE. It is more for tasks that are meant to be run manually. Since this is a helper, `group` is not necessary. Also `description`